# print(a)

# when we write above code than it shows that a is not define it is called exception
print("------------------------------------------------------------------------------------")
try:
    print(a)
except:
    print("something went wrong")

print("------------------------------------------------------------------------------------")
